cmd/nerdctl/compose.go

```diff
@@ -42,6 +42,7 @@ func newComposeCommand() *cobra.Command {
 		newComposeUpCommand(),
 		newComposeLogsCommand(),
 		newComposeConfigCommand(),
+		newComposeCopyCommand(),
```

Comment:
Could you also update https://github.com/containerd/nerdctl/blob/main/docs/command-reference.md ?